packages/shared/src/mock/metadata.ts

```diff
@@ -151,6 +151,17 @@ export const mockMetadata = makeNxGroup(mockFilepath, 'NXroot', {
         }),
       ],
     }),
+    makeNxGroup('nexus_entry_no_default', 'NXentry', {
+      defaultPath: undefined,
+      children: [
+        makeNxGroup('ignore_me', 'NXprocess'),
+        makeNxDataGroup('spectrum', {
+          signal: makeNxDataset('oneD', intType, [41]),
+          axes: { X: makeNxDataset('X', floatType, [41]) },
+          axesAttr: ['X'],
+        }),
+      ],
+    }),
```

Comment:
I've just moved this group up and renamed it to get this:

![image](https://user-images.githubusercontent.com/2936402/140064709-3352dd5f-c644-4f8e-81e3-d9c4a6e91982.png)

I've also added an `NXprocess` entity to check that it is correctly skipped when looking for the default plot.